app/components/Dashboard/Cards/WhatDifference.tsx

```diff
@@ -27,43 +35,43 @@ const Highlight: React.FC<React.PropsWithChildren> = ({ children }) => (
   <span className="text-green-500 font-semibold">{children}</span>
 )
 
-const Cards = () => (
+const Cards: React.FC<CardsProps> = ({ household }) => (
   <div className="flex flex-wrap gap-6">
-    <Card title="Money saved" figure="£30,000">
-      <p>On housing costs over 10 years, compared to conventional ownership</p>
+    <Card title="Money saved" figure={`£${Math.round(household.socialValue.moneySaved).toLocaleString()}`}>
```

Comment:
If you're not familiar with it, have a look at this API - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat